lighthouse-core/computed/metrics/lantern-first-contentful-paint.js

```diff
@@ -24,52 +26,109 @@ class LanternFirstContentfulPaint extends LanternMetric {
   }
 
   /**
-   * @param {Node} dependencyGraph
-   * @param {LH.Artifacts.TraceOfTab} traceOfTab
-   * @return {Node}
+   *
+   * @param {Node} graph
+   * @param {number} fcpTs
+   * @param {function(NetworkNode):boolean} networkFilter
+   * @param {function(CPUNode):boolean=} cpuFilter
```

Comment:
does `{(function(CPUNode):boolean)=}` work? `=` isn't supposed to be applicable to return types, but with tsc you never know, so it ends up hard to read

Reply:
done 👍